fields/field.textbox.php

```diff
@@ -310,6 +311,29 @@ public function displaySettingsPanel(XMLElement &$wrapper, $errors = null) {
 			$label->setAttribute('class', 'column');
 			$columns->appendChild($label);
 
+			$input = Widget::Input(
+				"fields[{$order}][handle_unique]",
+				'no', 'hidden'
+			);
+			$columns->appendChild($input);
+
+			$input = Widget::Input(
+				"fields[{$order}][handle_unique]",
+				'yes', 'checkbox'
+			);
+
+			if ($this->get('handle_unique') == 'yes') {
+				$input->setAttribute('checked', 'checked');
+			}
+
+			$label = Widget::Label(
+				__('%s Handles are unique', array(
+					$input->generate()
+				))
+			);
+			$label->setAttribute('class', 'column');
+			$columns->appendChild($label);
+
```

Comment:
Here's how I deal with checkbox that do not have value (it's 'on' by default).
https://github.com/Solutions-Nitriques/oembed_field/blob/master/fields/field.oembed.php#L298